contributed_definitions/nyaml/NXbias_spectroscopy.yaml

```diff
@@ -2,113 +2,75 @@ category: base
-      section). After the initial z averaging time, if Z-Controller to Hold is
-      selected in the Advanced section, the z-Controller is set to hold and the tip is
-      placed at the previously averaged z position (plus z offset).
-  sw_filter_type(NX_CHAR):
-    doc: |
-      Select a filter to smooth the displayed data. When saving, if any filter is selected,
-      filtered data are saved to file along with the unfiltered data.
-  sw_filter_order(NX_NUMBER):
-    doc: |
-      Filter order of a dynamic filter or width (in number of points) for the Gaussian
-      filter.
-  sw_filter_cutoff_frq(NX_NUMBER):
-    doc: |
-      Cutoff frequency for dynamic filters.
-  z_offset(NX_NUMBER):
+      The time that is needed to stabilize the prob position at a desired distance from 
+      the subject of scanning. 
+
+  Z_offset(NX_NUMBER):
     unit: NX_LENGTH
     doc: |
-      Offset added to the initial averaged position Zaver before starting to sweep.
-      This parameter is disabled when Z-Controller to Hold is deselected in the
-      Advanced section. The LED “Alt” next to the Z offset indicates if an alternate
-      Z-controller setpoint is enabled.
+      Initial position of the probe before starting the sweep.
   settling_time(NX_NUMBER):
     unit: NX_TIME
     doc: |
-      Time to wait after changing the bias to the next level and before
+      Time to wait after changing the bias, in the sweep process, to the next level and before
       starting to acquire data.
-  first_settling_time(NX_NUMBER):
+  DATA(NXdata):
     doc: |
-      No doc yet.
-  end_settling_time(NX_NUMBER):
-    unit: NX_TIME
-    doc: |
-      Time to wait after the sweep has finished and the bias is ramped to
-      the initial value.
-  z_control_time(NX_NUMBER):
-    unit: NX_TIME
-    doc: |
-      Time during which the Z-Controller is enabled once a sweep has finished.
-      When averaging multiple sweeps (i.e. Sweeps>1), the Z-Controller is enabled
-      for this duration between each sweep. After the last sweep, it will wait the
-      specified time before continuing a running scan. This ensures each sweep
-      reliably starts from the same position. This parameter is disabled when
-      Z-Controller to Hold is deselected in the Advanced section.
-  max_slew_rate(NX_NUMBER):
-    doc: |
-      Maximum rate at which the bias changes (before, during and after sweeping).
-      (V/s)
-  backward_sweep(NX_BOOLEAN):
-    doc: |
-      Select whether to measure the backward (return) sweep or the forward only.
-  z_ccontroller_hold(NX_BOOLEAN):
-    doc: |
-      Select whether to set the Z-Controller on hold (i.e. disabled) during the sweep.
-      It is selected by default. When deselected, Z-offset and Z control time parameters
-      are disabled.
+      Use this group to store the data from several channels.
+
+      The data I(V) and dI/dV(V) wrt the forward sweep or backward sweep should be stored in 
```

Comment:
Please spell out wrt. It is also a bit unclear what you mean by with respect to the forward sweep? Shouldn't I(V) and dI/dV(V) be stored here independent of the sweep direction?